### DISCUSSION TITLE
Add option to select default date / time visualization

### DISCUSSION THREAD

**Discussion by ghost** [grafana/grafana#57584] [Dashboards, Panels & Visualizations]
It would be very useful (and time saving) to have an option to select the default format for date and time (`YYYY-MM-DD` or `DD-MM-YYYY`, `HH:MM:SS` or `HH:MM` etc...) for the dashboard or the whole organization.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
